pkg/cli/gptscript.go

```diff
@@ -34,6 +35,7 @@ type GPTScript struct {
 	CacheOptions
 	OpenAIOptions
 	DisplayOptions
+	Color         *bool  `usage:"Use color in output (default true)" default:"true"`
```

Comment:
nit: can this just be a bool instead of *bool?

Reply:
I was following the example of the `--cache` flag, which you can see here: https://github.com/gptscript-ai/gptscript/blob/fd7e77df85168b79788629e9bd1000cab21c2dc5/pkg/cache/cache.go#L21

It uses a bool pointer. I noticed that when I had just a bool, the `default:"true"` struct tag didn't do anything, but it worked as soon as I made it a pointer.

Reply:
Also I just noticed that the description for `--cache` is the opposite of what it's supposed to be, lol